pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -0,0 +1,424 @@
+package testint
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_SecurityIntegrations(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	acsURL := fmt.Sprintf("https://%s.snowflakecomputing.com/fed/login", testClientHelper().Context.CurrentAccount(t))
```

Comment:
nit-pick: These two seem to be good candidates to move the to helpers (please do it with PR with the rest of security integrations)

Reply:
Added a TODO.